src/modules/sensors/sensors.cpp

```diff
@@ -167,12 +167,12 @@ class Sensors : public ModuleBase<Sensors>
 	int		_vcontrol_mode_sub;		/**< vehicle control mode subscription */
 	int 		_params_sub;			/**< notification of parameter updates */
 
-	orb_advert_t	_sensor_pub;			/**< combined sensor data topic */
-	orb_advert_t	_battery_pub[BOARD_NUMBER_BRICKS] = {nullptr};			/**< battery status */
```

Comment:
I strongly prefer to have the initialization in the definition. It makes it much easier to see that everything is covered and avoids the initialization order problems.

I also have a slight preference for c++11 uniform initialization which has some advantages for non primitive data types. https://www.youtube.com/watch?v=yKYcybUEX3I&feature=youtu.be&t=23m

All that is kind of irrelevant though because we don't have written coding standards, so for now I'd say consistency wins. Doing a pass to make sure everything is properly initialized in the style we prefer could make for good introductory CQWG tasks.